### DISCUSSION TITLE
The "iOS SDK" field

### DISCUSSION THREAD

**Discussion by codeandcode123** [airsdk/Adobe-Runtime-Support#3185] [Q&A]
I'm trying to publish an iOS app using Adobe Animate on a Mac. What should be entered into the "**iOS SDK**" field? 

And how to "Please edit entitlement section in the descriptor xml file"?

Thanks!

![Screenshot 2024-04-18 at 3 38 05 PM](https://github.com/airsdk/Adobe-Runtime-Support/assets/129521876/00033ec3-1f39-4d93-b992-16f625520b39)

**Comment by lconical** [comment]
Also having this issue...

**Comment by marchbold** [comment]
If you have installed Xcode and the iOS SDK in the default location you shouldn't have to enter anything here, however the default location is something like:

```
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.4.sdk
```

Regarding Entitlements and InfoAdditions you add these in your application descriptor. If you are using any ANE's the documentation for these should list the required additions, or if you are using `apm` they will get inserted as part of the `generate app-descriptor` command.

**Comment by lconical** [comment]
Were getting the following error when referencing our default sdk path:

The path to platform sdk should end with any one of the (iPhoneOSx.x | iPhoneSimulatorx.x | AppleTVSimulatorx.x | AppleTVOSx.x) format to avoid any unintended behavior.
ld: framework not found AppTrackingTransparency

Googling around it seems to be something to do with IDFA ANE, and xcode 12. we're currently running xcode 10.1 on Mojave for various reasons and are unable to upgrade to 12 on Mojave. 

Weve also followed this tutorial and tried downloading your SDKS here while referencing the downloaded SDK in animate, with no luck and the same error message " framework not found AppTrackingTransparency" when attempting to test on our IOS devide:

https://resources.airnativeextensions.com/ios/

https://docs.airnativeextensions.com/docs/tutorials/ios-sdk-custom

Any advice appreciated...

**Reply by ajwfrost** [reply]
AppTrackingTransparency was introduced in iPhoneOS SDK 14.5 so you would need to update to that in order to make use of this ANE...

Alternatively you could try omitting the "platformsdk" argument (i.e. removing any value from this "iOS SDK" field) and then the AIR SDK stub framework would be used instead. Which may still work given you're using older versions of the linker etc...
